scripts/scripts.js

```diff
@@ -42,6 +44,63 @@ async function loadFonts() {
   }
 }
 
+/**
+ * Opens an iframe with the video when clicking on anchor tags.
+ * @param {Element} element container element
+ */
+
+function openVideoIframe(element) {
```

Comment:
As this decorates/adds listeners rather than "opens", can you rename it so something like decorateVideoLinks or similar?